Gemfile

```diff
@@ -61,7 +61,7 @@ gem "unicorn" # Use Puma as the app server
 gem "unicorn-worker-killer"
 
 # Making other files
-gem "axlsx", "~> 3.0.0.pre" # Write Excel files (OrganizationExports), on pre b/c gem isn't otherwise updated
+# gem "axlsx", "~> 3.0.0.pre" # Write Excel files (OrganizationExports), on pre b/c gem isn't otherwise updated
```

Comment:
Requires an old versions of nokogiri. Minor adjustment made in #1072 - the issue on axlsx that's referenced there has a comment recommending [xlsxtream](https://github.com/felixbuenemann/xlsxtream) - but [spreadsheet](https://github.com/zdavatz/spreadsheet) might be better since it's more widely used.